File: code_sample/php/mytest.php
<?php

function toUpper($str) {
    $new_str = "";
    foreach(str_split($st) as $chr) {
        $new_str = $new_str . strtoupper($str);
    }
    return $new_str;
}

class FindInvitationsTest extends \PHPUnit_Framework_TestCase 
{
    public function testToUpper()
    {
        $this->assertEquals(toUpper('Hello World'), 'HELLO WORLD');
    }
}

File: code_sample/python/mytest.py
import unittest

def toUpper(str):
    new_str = ""
    for chr in str:
        new_str.append(chr.upper())

    return str

class MyTest(unittest.TestCase):

    def testToUpper(self):
        self.assertEqual(toUpper('Hello world'), 'HELLO WORLD')

if __name__ == '__main__':
    unittest.main()
